.github/workflows/e2e-tests-ci.yml

```diff
@@ -95,15 +136,24 @@ jobs:
     needs:
       - cypress-check
       - playwright-check
+      - generate-build-variables
     defaults:
       run:
         shell: bash
         working-directory: e2e-tests
+    env:
+      AUTOMATION_DASHBOARD_URL: "${{ secrets.MM_E2E_AUTOMATION_DASHBOARD_URL }}"
+      AUTOMATION_DASHBOARD_TOKEN: "${{ secrets.MM_E2E_AUTOMATION_DASHBOARD_TOKEN }}"
+      MM_LICENSE: "${{ secrets.MM_E2E_LICENSE_ONPREM }}"
```

Comment:
I think this is fine for `smoketests` for there's no test related to license delettion. However, for full E2E tests, I'd suggests to change loading license to file-based instead of env variable so it will address https://mattermost.atlassian.net/browse/CLD-6926

Reply:
Ah, I wasn't aware of this issue. I'll adapt this PR accordingly, thanks!

Reply:
This is done, the `MM_VARIABLE` is not passed to the server container anymore, and instead is used in a `mmctl license upload-string` command at server preparation time. This should leave the E2E tests free to remove the license as they please.

Reply:
Thanks Mario!

(reopening to have on the same thread)

One last which I missed to comment, can we change the name of the secret to be more descriptive to something like `MM_E2E_TEST_LICENSE_ONPREM_ENT` where:
- `TEST` indicates service environment, other value is `PROD`
- `ENT` indicates enterprise plan, other values are professional (`PRO`) and starter (`STA`?).

Since this is org env, it would be easier to identify for others to use in their testing depending on what license they intend to load into the test server.

Reply:
Great point. I renamed the variable to `MM_E2E_TEST_LICENSE_ONPREM_ENT` as you suggested, thanks!